lib/http/converter/utc_date_nullable_converter.dart

```diff
@@ -5,8 +5,8 @@ class UTCDateNullableConverter implements JsonConverter<UTCDate?, String?> {
   const UTCDateNullableConverter();
 
   @override
-  UTCDate? fromJson(String? json) => json != null ? UTCDate(json) : null;
+  UTCDate? fromJson(String? json) => json != null ? UTCDate(DateTime.parse(json).toUtc()) : null;
 
   @override
-  String? toJson(UTCDate? object) => object?.value;
+  String? toJson(UTCDate? object) => object?.value.toString();
```

Comment:
are you sure, it will be show a date time with `Z` offset? How about this value come from the input of user?

Reply:
UTCDate support format utc

Reply:
yes, but when you convert `toJson`, are you sure it is UTC format.

Reply:
I use `isUtc` function in `DateTime` to test it.